CryptorRSA.xcodeproj/xcshareddata/xcschemes/CryptorRSA-iOS.xcscheme

```diff
@@ -27,7 +27,23 @@
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES">
+      <TestPlans>
+         <TestPlanReference
+            reference = "container:Tests/CryptorRSA-iOSTests/CryptorRSA-iOSTests.xctestplan"
+            default = "YES">
+         </TestPlanReference>
+      </TestPlans>
```

Comment:
I must admit I have not yet used TestPlans, but they look really useful!  How can I trigger them?  I can't seem to find a way to exercise them.

Reply:
When using the .xcodeproj it ties into the scheme. In the scheme 'Test' step, you can choose what "test plan" to run (instead of choosing what test target to run).

One of my favorite features is the "random" order - it just helps to pinpoint any tests that accidentally have a dependency on run order (very helpful if you have multiple people contributing to the same repo).

Otherwise, they just help keep things a little bit cleaner (I wonder if Apple's long term plan is to eliminate to the need to have a Test target as part of the project build??)